spec/factories/restaurant_factory.rb

```diff
@@ -49,5 +49,18 @@
       max_delivery_time_min { 720 }
       accepts_10bis { false }
     end
+    trait :invalid_record do
```

Comment:
this is a bit problematic... if you will change one of the validations it will cause to a flaky test that will fail randomly (no one will remember to change this trait when he changes a validation)